src/main/java/com/google/cloud/tools/appengine/api/devserver/RunConfiguration.java

```diff
@@ -72,4 +72,6 @@
   String getDefaultGcsBucketName();
 
   String getJavaHomeDir();
+
+  Boolean getClearDatastore();
```

Comment:
We want default behavior to be null, so we always delegate to gcloud defaults (which can be true or false)